.gitignore

```diff
@@ -82,13 +82,25 @@ Podfile.lock
 !**/ios/**/default.pbxuser
 !**/ios/**/default.perspectivev3
 
+# macOS-related
+**/macos/**/Pods/
+**/macos/**/xcuserdata
+**/macos/**/Flutter/ephemeral
+
+# Windows-related
+**/windows/flutter/ephemeral/
+
+# Linux-related
+**/linux/flutter/ephemeral/
+
 # Generated files
 *.dart.js
 *.info.json
 *.js
 *.js_
 *.js.deps
 *.js.map
+*.wasm
```

Comment:
Is it not enough to ignore the build folders?

Reply:
Will these files always be present in build folders? I was not sure so added them specifically and added exceptions for the `sql-wasm.wasm` for the sample app.

Reply:
I would expect that when these files are not in the build folder they are useful to keep. But I would have to see which files these are and why they are there.